components/TopBarSmallScreen.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+    <v-app-bar elevation="0">
+        <template v-slot:prepend>
+            <v-icon color="accent" :icon="useNavDrawerOpen().value ? 'mdi-close' : 'mdi-menu'" @click="toggleNavDrawer"></v-icon>
+        </template>
+
+        <TitleAndLogo />
+
+        <template v-slot:append>
+            <ClientOnly>
+                <ConnectButton />
+            </ClientOnly>
+        </template>
+    </v-app-bar>
+</template>
```

Comment:
Newline at end of files plz

Reply:
I'd like to see these components combined into one so that the logic / template is all contained in the same place.

You can change the structure with logic directives like this:

```vue
<template v-if="condition == 1"><template>
<template v-else-if="condition == 1"><template>
<template v-else></template>
```

v-if and related directives work on any component, not just template, but here you could have a template logic structure

Reply:
Ok so can combine topbar and topbarsmallscreen into one file top bar and then use that template logic switch like youre saying. Will do